lib/Ohmbrewer_Menu.cpp

```diff
@@ -0,0 +1,156 @@
+#include "Ohmbrewer_Menu.h"
```

Comment:
Discussed with @azyth, will create a subfolder for menu classes to prevent clutter.